### DISCUSSION TITLE
我想更改风场和台风的样式 可是改代码并不生效 是为什么呢？

### DISCUSSION THREAD

**Discussion by w132290293** [zouyaoji/vue-cesium#453] [Q&A]
大佬您好，我更改了model里的vue-cesium的components组件代码 是lib和es里的  .js   .js.map  .mjs都改了 可是还不生效，我用vite的--force重新构建库也不生效，是需要其他操作吗？还是说我更改的位置不对

**Comment by zouyaoji** [comment]
具体咋改的你贴出来一下哈
